src/utils.ts

```diff
@@ -52,6 +54,10 @@ export default class utils {
         return chains[chain].getAddress(privateKey)
     }
 
+    static signTransaction(chain: string, privateKey: string, transaction: any): object {
```

Comment:
Why was this added?

Reply:
since this util does all the offline functions, like creating addresses, signing messages, I figured we would use it to sign transactions too?